app/javascript/onboarding/components/FollowTags.jsx

```diff
@@ -74,30 +74,45 @@ class FollowTags extends Component {
           credentials: 'same-origin',
         }),
       ),
-    ).then(_ => {
+    ).then((_) => {
       const { next } = this.props;
       next();
     });
   }
 
+  renderFollowCount() {
+    const { selectedTags } = this.state;
+    let followingStatus;
+    if (selectedTags.length < 2) {
+      followingStatus = `${selectedTags.length} topic selected`;
+    } else {
+      followingStatus = `${selectedTags.length} topics selected`;
+    }
+
+    const klassName =
```

Comment:
```suggestion
    const classStyle =
```
Just a suggestion, it felt really weird to see `klass` in JavaScript 😱

Reply:
Updated 😆